packages/foreman/rubygem-jwt/rubygem-jwt.spec

```diff
@@ -1,69 +1,100 @@
+# Generated from jwt-2.1.0.gem by gem2rpm -*- rpm-spec -*-
```

Comment:
JWT is a special case. We need to ship 1.2.1 in the non-SCL context but we also have a 2.1.0 version as `tfm-rubygem-jwt` in packages/foreman for the SCL context.

Reply:
Can you revert the changes to this file (and the gems)?